src/ecmascript_simd.js

```diff
@@ -474,13 +474,13 @@ if (typeof SIMD.Bool32x4.select === "undefined") {
   }
 }
 
-if (!Object.hasOwnProperty(SIMD.Bool32x4.prototype, 'valueOf')) {
+if (!SIMD.Bool32x4.prototype.hasOwnProperty('valueOf')) {
```

Comment:
Do you think this sort of change is required, or would you be happy to see this patch go in as is